src/core.js

```diff
@@ -1388,10 +1373,16 @@ define([
 
         if (this._propertiesMug) {
             this._propertiesMug.teardownProperties();
+            try {
+                this._propertiesMug.validate();
+            } catch (err) {
+                // ignore error
```

Comment:
At the very least maybe log it as a warning to the console? Unless this error is completely not important